runtimes/manta/runtime/src/lib.rs

```diff
@@ -107,8 +106,8 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	//   `spec_version`, and `authoring_version` are the same between Wasm and native.
 	// This value is set to 100 to notify Polkadot-JS App (https://polkadot.js.org/apps) to use
 	//   the compatible custom types.
-	spec_version: 2,
-	impl_version: 1,
+	spec_version: 3,
+	impl_version: 2,
```

Comment:
In my opinion, most of time, we don't need to bump `impl_version`.
Reference: https://substrate.dev/docs/en/knowledgebase/runtime/upgrades

Reply:
agreed. will revert it back.